core/codegen/src/attribute/route/mod.rs

```diff
@@ -296,6 +296,38 @@ fn sentinels_expr(route: &Route) -> TokenStream {
     quote!(::std::vec![#(#sentinel),*])
 }
 
+fn schemas_expr(route: &Route) -> TokenStream {
+    let ret_ty = match route.handler.sig.output {
+        syn::ReturnType::Default => None,
+        syn::ReturnType::Type(_, ref ty) => Some(ty.with_stripped_lifetimes())
+    };
+
+    let generic_idents: Vec<_> = route.handler.sig.generics
+        .type_params()
+        .map(|p| &p.ident)
+        .collect();
+
+    let eligible_types = route.guards()
```

Comment:
We don't need any of this. We simply need the top-level types passed to the resolve macro. This only exists in sentinels due to embedded discovery.